install.sh

```diff
@@ -8,7 +8,12 @@ HTTPD_WELCOME="/etc/httpd/conf.d/welcome.conf"
 INSTALL_PKGS="highlight httpd nss_wrapper gettext";
 
 # Setup necessary packages
-yum -y install epel-release && yum -y install ${INSTALL_PKGS};
+yum -y install epel-release && yum -y install ${INSTALL_PKGS} bzip2;
+yum -y update;
+rpm -ivh https://kojipkgs.fedoraproject.org//packages/http-parser/2.7.1/3.el7/x86_64/http-parser-2.7.1-3.el7.x86_64.rpm && yum -y install nodejs
```

Comment:
Is this really a good idea? Shouldn't we be using SCL here?

Reply:
This was a problem in CentOS with http-parser being removed from EPEL, see https://bugzilla.redhat.com/show_bug.cgi?id=1481470
It should be fixed now, see also https://github.com/fabric8-analytics/fabric8-analytics-worker-base/pull/10

Reply:
@jyasveer so this line can be removed.